docs/DesignDocs/CustomAttestation.md

```diff
@@ -927,9 +965,24 @@ size_t params_size = sizeof(params);
 oe_claim_t claims = { ... };
 size_t claims_size = ...;
 
+/* Receive the evidence format ids that the verifier supports */
+recv(VERIFIER_SOCKET_FD, evidence_format_ids, evidence_format_id_length, 0);
+
+/* Get registered attester format ids and find a common format */
+oe_get_registered_attester_format_ids(*format_ids, &format_ids_length);
```

Comment:
Does it make this search functionality part of attest_plugin.c? This would avoid having to return the list of format_ids.

Reply:
The search functionality can be performed by calling oe_get_evidence() and check if it returns OE_NOT_FOUND.

This code snippet is to demonstrate the usage of oe_get_registered_attester_format_ids.

---
In reply to: [365373127](https://github.com/openenclave/openenclave/pull/2402#discussion_r365373127) [](ancestors = 365373127)

Reply:
Who will be using this new set of functions?

Reply:
Discussed offline.  To update doc.